docs/install/docker.md

```diff
@@ -11,6 +11,11 @@ If you are already set up with Docker, you may wish to refer to the
 
 ## First Time Installation
 
+### IMPORTANT INFORMATION FOR WINDOWS USERS
```

Comment:
A couple things here. I think inserting a header here makes the flow confusing. There's no indication when the text that applies to all users begins. I suggest moving this above the `First Time Installation` heading and adding a link to it in the table of contents as well. Also I suggest naming it something more specific like `Windows Git Config` and not uppercasing all the letters. No need to yell in our documentation ;)